docs/guide/custom-ui.md

```diff
@@ -250,6 +250,79 @@ Do not use core actions as your `action` property value - this would lead to unp
 * For backward compatibility, this method can also be called with the `moduleName` passed as the first argument and the object as the second, but this is discouraged.
 :::
 
+## Toggling the admin-bar visibility
```

Comment:
"Toggling the visibility of the admin bar" might read better?

Reply:
Changed with a hyphen added.